### ISSUE TITLE
Look into Removing PneumaticCraft

### ISSUE THREAD

**Issue by UndeadZeratul** [UndeadZeratul/ThereWillBeBlood#140]
Every time I think about it, I can't justify its pressure mechanics against its place in the progression.  It seems too arbitrary, and if it weren't for its oil processing, it'd be completely pointless.  Hell, even the fact that by the time you get to make the oil refinery, you can already melt down stuff in the high oven which gives a higher melting temperature than any of the oil products can.  The more I think about it, the less I want it.

If I'm going to get rid of it, it's going to need to be removed before v1.0.0, since it involves worldgen.  Also, since the oil and whatnot are used in the Smeltery as fluids, I'm going to have to make sure I get rid of those references as well.  I'm thinking, though, that I can instead use Ignatius and Vulcanite in their place as Smeltery fuels.  I'll write up a separate issue for new fuel ideas.

**Comment by UndeadZeratul** [comment]
A part of this will involve removing existing tweaks regarding PneumaticCraft content, such as using Oil and its refined products as fuel in the Smeltery.  As such, PneumaticCraft cannot be removed until #141 is complete, and is therefore blocked by it.

**Comment by UndeadZeratul** [comment]
After closing #141, I've removed the block on this issue.
